README.md

```diff
@@ -10,6 +10,11 @@ To quickly get started with Amplify (Gen 2), please visit [AWS Amplify (Gen 2) Q
 npm create amplify@latest
 ```
 
+## Package Manager Support
+
+- NodeJS 20 or later version is required for Yarn because of the NodeJS [loader bug](https://github.com/nodejs/node/pull/43772) causing yarn [chain loader error](https://github.com/yarnpkg/berry/issues/4694).
+- Yarn PnP is not supported. Run `yarn config set nodeLinker node-modules` to use "node-modules".
```

Comment:
This still doesn't have a list of package managers that are supported.

Reply:
awhhhh....good catch! done 5c744d5c4e